tests/test_main.py

```diff
@@ -23,57 +26,80 @@
 file_path = os.path.join(dir_path, "input/ARL.json")
 
 
-def test_main(mocker: MockerFixture):
-    """Unit test for main() function in main.py"""
-    mock_cmd_arguments = MagicMock()
-    patch_run_rufas = mocker.patch(
-        "main.run_rufas"
-    )
-    patch_parse_gnu_args = mocker.patch(
-        "main.parse_gnu_args", return_value=mock_cmd_arguments
-    )
-
-    main()
+@pytest.mark.parametrize(
+    "format_option, no_graphics, graphics_dir, verbose, clear_output, exclude_info_maps, only_run_validation",
+    [
+        ("verbose", False, "graphics", True, True, True, True),
+        ("basic", True, "custom_graphics", False, False, False, False),
+        ("block", True, "graphics", False, True, False, False),
+        ("inline", False, "custom_graphics", True, False, False, False),
+        ("verbose", True, "graphics", False, False, True, False),
+    ],
+)
+def test_main(
+    format_option: str,
+    no_graphics: bool,
+    graphics_dir: str,
+    verbose: bool,
+    clear_output: bool,
+    exclude_info_maps: bool,
+    only_run_validation: bool,
+) -> None:
+    with patch("main.parse_gnu_args") as mock_parse_gnu_args:
```

Comment:
```suggestion
    with patch("main.parse_gnu_args") as mock_parse_gnu_args, \
            patch("main.run_rufas") as mock_run_rufas:
```
It is possible to have multiple patches in the same context manager, instead of nesting them. It isn't really needed here, but might be helpful in the future if more patches need to be added.

Reply:
I think it is, but I'm to keep the code as it is here.